cmd/loom/loom.go

```diff
@@ -437,6 +438,51 @@ func newRunCommand() *cobra.Command {
 	return cmd
 }
 
+const contractInfoCommandExample = `
+loom contract default:0x81ee596ba88eF371a51d4B535E07cB243A8C692d
+`
+
+type contractInfo struct {
+	Name    string
+	Address string
+	Owner   string
+}
+
+func contractInfoCommand() *cobra.Command {
+	var flags cli.ContractCallFlags
+	cmd := &cobra.Command{
+		Use:     "contract [ChainID:Address]",
+		Short:   "Get contract information by address",
+		Args:    cobra.MinimumNArgs(1),
+		Example: contractInfoCommandExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			addr, err := cli.ParseAddress(args[0], flags.ChainID)
```

Comment:
Use `cli.ResolveAddress` so that it's possible to pass in a contract name or a contract address.